lib/super_digit.rb

```diff
@@ -1,15 +1,20 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n) - dependent on length of number
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
Note that inject is O(n) and it's nested in recursion, so this will be worse than O(n).